code/src/components/app-range-slider/AppRangeSlider.test.tsx

```diff
@@ -0,0 +1,73 @@
+import { render, screen } from "@testing-library/react";
+
+import AppRangeSlider from "@/components/app-range-slider/AppRangeSlider";
+
+import typeIntoInput from "@/utils/type-into-input/typeIntoInput";
+
+const mockHandleChange = jest.fn();
+
+describe("AppRangeSlider", () => {
+  describe("with default props", () => {
+    test("uses default props correctly", () => {
+      render(<AppRangeSlider />);
```

Comment:
use `renderComponent` own utility with `props` as you created early with some other tests
And use it once for other places with `renderWithProviders` instead `render`. Follow single approach for render component

Reply:
Thanks, fixed